packages/liveblocks-client/.eslintrc.js

```diff
@@ -0,0 +1,56 @@
+module.exports = {
+  root: true,
+  parser: "@typescript-eslint/parser",
+  parserOptions: { project: ["./tsconfig.json"] },
+  plugins: ["@typescript-eslint"],
+  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+  rules: {
+    // ----------------------------------------------------------------------
+    // NOTE: Only temporarily turned off!
+    // These checks are still GOOD IDEAS to re-enable later on, but for right
+    // now they're too noisy to enable.
+    // ----------------------------------------------------------------------
+    "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-non-null-asserted-optional-chain": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
+
+    // -------------------------------
+    // Not interested in these checks:
+    // -------------------------------
+    "@typescript-eslint/no-empty-function": "off",
+    "@typescript-eslint/no-inferrable-types": "off",
+    "no-constant-condition": "off",
+
+    // ------------------------
+    // Customized default rules
+    // ------------------------
+    "@typescript-eslint/no-unused-vars": [
+      "warn",
+      // Unused variables are fine if they start with an underscore
+      { argsIgnorePattern: "^_.*", varsIgnorePattern: "^_.*" },
+    ],
+
+    // -------------------------------
+    // Custom syntax we want to forbid
+    // -------------------------------
+    "no-restricted-syntax": [
+      "error",
+      // {
+      //   selector: "PrivateIdentifier",
+      //   message:
+      //     "Avoid private identifiers to reduce bundle size. Instead of using `#foo`, prefer using `private _foo`.",
+      // },
```

Comment:
cc @GuillaumeSalles after you merge https://github.com/liveblocks/liveblocks/pull/106, we can uncomment this to enforce that we avoid private fields/methods in the future automatically.

Reply:
This was fixed by #135.